packages/pyright-internal/src/analyzer/docStringConversion.ts

```diff
@@ -421,8 +424,13 @@ class DocStringConverter {
     }
 
     private _beginBacktickBlock(): boolean {
-        if (this._currentLine().startsWith('```')) {
-            this._appendLine(this._currentLine());
+        const match = this._currentLine().match(CodeBlockStartRegExp);
+        if (match !== null) {
+            this._blockIndent = this._currentIndent()
+
+            // Remove indentation and preserve language tag.
+            this._appendLine('```' + match[1])
```

Comment:
Match group would be null if the language tag is missing?

Reply:
match returns an empty string if a capture group is empty
```
> '    ```'.match(/\s*```(\w*)/)[1]
''
```